src/Kiota.Builder/Writers/Php/CodeMethodWriter.cs

```diff
@@ -90,6 +90,19 @@ public override void  WriteCodeElement(CodeMethod codeElement, LanguageWriter wr
                 AssignPropertyFromParameter(parentClass, currentMethod, CodeParameterKind.PathParameters, CodePropertyKind.PathParameters, writer);
                 AssignPropertyFromParameter(parentClass, currentMethod, CodeParameterKind.RawUrl, CodePropertyKind.UrlTemplate, writer);
             }
+            var pathParametersProperty = parentClass.GetPropertyOfKind(CodePropertyKind.PathParameters);
+            if (currentMethod.IsOfKind(CodeMethodKind.Constructor, CodeMethodKind.ClientConstructor) &&
+                parentClass.IsOfKind(CodeClassKind.RequestBuilder) &&
+                currentMethod.Parameters.Any(x => x.IsOfKind(CodeParameterKind.Path)))
+            {
+                writer.WriteLine($"$urlTplParams = $pathParameters;");
```

Comment:
why the hardcoded names instead of using the property and parameter names since you already have them?